bundle/artifacts/build.go

```diff
@@ -80,6 +81,7 @@ func (m *build) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
 		}
 
 		if a.Type == "whl" && a.DynamicVersion && cacheDir != "" {
+			b.Metrics.AddBoolValue(protos.DynamicVersionIsSet, true)
```

Comment:
Does this mean multiple entries if you have multiple artifacts with this setting?

Reply:
Thanks, yeah that works, as long as this is documented as supported, both in the protos and the function signature.

Reply:
Discussed with @shreyas-goenka offline: there is no expectations that key should be unique in this BoolMapEntry structure thus duplicated values will work correctly since it's just an array of objects from proto perspective with no special meaning to any of the fields